flink/flink-1.14/src/main/java/io/tidb/bigdata/flink/connector/utils/TiDBRowConverter.java

```diff
@@ -142,12 +142,8 @@ public Row toTiRow(RowData rowData, boolean ignoreAutoincrementColumn) {
       DataType type = tiColumnInfo.getType();
       if (rowData.isNullAt(i)) {
         tiRow.setNull(i);
-        // delete RowKind only need pk not null
+        // delete RowKind can be null for TiCDC may close old value
```

Comment:
What's the meaning of this sentence

Reply:
if TiCDC close old value, delete row will only have one column called `HandleKey.`
then we should not throw exception for the not null column,because we can also execute delete with `HandleKey`

Reply:
I will optimize the comments to make it easier to understand